tools/listblocks/main.go

```diff
@@ -140,7 +141,7 @@ func printMetas(metas map[ulid.ULID]*block.Meta, deletedTimes map[ulid.ULID]time
 	fmt.Fprintln(tabber)
 
 	for _, b := range blocks {
-		if !cfg.showDeleted && !deletedTimes[b.ULID].IsZero() {
+		if !cfg.showDeleted && deleteMarkerDetails[b.ULID].DeletionTime != 0 {
```

Comment:
We could change `DeletionTime` to `time.Time` and provide correct marshal/unmarshal functions.